src/explore-education-statistics-admin/src/pages/release/edit-release/content/ReleaseContextActionTypes.ts

```diff
@@ -80,6 +81,14 @@ export type UpdateContentSection = {
   };
 };
 
+export type UpdateBlockComments = {
+  type: 'UPDATE_BLOCK_COMMENTS';
```

Comment:
At some future point, it might be better to put this string into a type somewhere, to make sure it's typesafe.

Reply:
This is type-safe. It's literally a type.

Reply:
Haha oh yeah :)